privaterelay/templates/includes/login.html

```diff
@@ -6,18 +6,27 @@
 {% block content %}
 
 <div class="user-area">
-	<firefox-apps></firefox-apps>
 	{% if request.user.is_anonymous %}
+	<a
+	 data-entrypoint="relay-sign-in-header"
+	 data-form_type="button"
+	 data-ga="send-ga-funnel-pings"
+	 data-event-category="Sign In Buttons"
+	 data-event-label="sign-in-btn-header"
+	 class="sign-up button"
+	 href="{% provider_login_url 'fxa' process='login' %}">{% ftlmsg 'nav-profile-sign-up' %}</a>
     <a
       data-entrypoint="relay-sign-in-header"
       data-form_type="button"
       data-ga="send-ga-funnel-pings"
       data-event-category="Sign In Buttons"
       data-event-label="sign-in-btn-header"
-      class="sign-in button btn-transparent"
+      class="mzp-c-button mzp-t-secondary mzp-t-dark home-sign-in"
```

Comment:
Added Protocol button classes here

Reply:
nit: Recommend using a different class name than `home-sign-in` that closer matches the [class naming convention](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md#class-naming-convention).